lib/teach-api.js

```diff
@@ -73,6 +73,20 @@ _.extend(TeachAPI.prototype, {
     }
 
     return req;
+  },
+  getAllData: function(callback) {
+    request('get', process.env.TEACH_API_URL || this.baseURL + "/api/clubs")
+      .accept('json')
+      .end(function(err, res) {
+        if(err) {
+          return callback(err);
+        }
+        try {
+          callback(null, JSON.parse(res.text));
```

Comment:
I believe you can actually just return `res.body` here, which contains the already-parsed response content.